templates/scenarios/notification-restify-js/.fx/projectSettings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "projectId": "92f9e2d5-2d2e-482c-9ac3-7ed261b3c4bf",
```

Comment:
Maybe a placeholder? Or just remove it?

Reply:
It is removed in template. Thanks.
@jayzhang, it seems the projectId should be generated in create project stage.